src/components/ArtifactsPreview/ArtifactsPreview.js

```diff
@@ -44,10 +53,17 @@ const ArtifactsPreview = ({ artifact }) => {
               ...prevState,
               { ...content, header: previewItem.header }
             ])
-            setIsError(false)
+            setError({
+              text: '',
+              body: ''
+            })
+            setShowError(false)
           })
-          .catch(() => {
-            setIsError(true)
+          .catch(err => {
+            setError({
+              text: `${err.response.status} ${err.response.statusText}`,
+              body: JSON.stringify(err.response)
```

Comment:
Let's prettifiy it using 2-space indents:
```suggestion
              body: JSON.stringify(err.response, null, 2)
```